Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -65,8 +65,8 @@ def check_xcode_up_to_date
         return unless MacOS::Xcode.installed? && MacOS::Xcode.outdated?
 
         message = <<-EOS.undent
-          Your Xcode (#{MacOS::Xcode.version}) is outdated
-          Please update to Xcode #{MacOS::Xcode.latest_version}.
+          Your Xcode (#{MacOS::Xcode.version}) is outdated.
+          Please update to Xcode #{MacOS::Xcode.latest_version} (or delete it).
```

Comment:
Basically: if you ever need Xcode, we'll tell you and fail to install otherwise.